Docs/source/Software/Tasks.rst

```diff
@@ -211,3 +211,30 @@ Yields
 Additional Considerations
     When modifying the idle task, it is important to not introduce any functionality that may affect other tasks. For example, the idle task should not pend 
     any mutexes, since this could block more important tasks from running.
+                   
+Log Info Task
+=============
+
+Purpose
+   This task logs the state of charge into the EEPROM every 3 seconds.
+
+Functionality
+   The log info task runs an infinite loop. Inside the loop, it sends percentage of charge left in the battery pack to the EEPROM using EEPROM_LogData and is   then delayed by OSTimeDly every 3 seconds. 
```

Comment:
```suggestion
   The log info task runs an infinite loop. Inside the loop, it sends percentage of charge left in the battery pack to the EEPROM using ``EEPROM_LogData()`` and is then delayed by ``OSTimeDly()`` every 3 seconds. 
```

Everything enclosed in the double backticks (``) gets formatted as code instead of regular text